Model/Request/Transactions.php

```diff
@@ -93,6 +93,7 @@ private function getBuyerParams(Quote $quote) {
         $params = [];
         if (($billing = $quote->getBillingAddress()) !== null) {
             $params = [
+                'is_registered' => $quote->getCustomer()->getId() ? true : false,
```

Comment:
is it safe to assume getCustomer() isn't null?

Reply:
It's safe, I tested it with a guest user and getCustomer a model object